test/integration/default/vbscript_spec.rb

```diff
@@ -18,5 +18,5 @@
 
 # ensure that we do not require a newline
```

Comment:
This comment implies that the property we were trying to test was explicitly that the user doesn't need to supply the line ending; but the change to the test implies they now do have to provide the line ending.